index.bs

```diff
@@ -765,6 +760,13 @@ requirements.</p>
  [[#parsing]], [[#rendering]] and [[#css-extensions]] in such a way that the rendered results are
  equivalent to what a full CSS supporting renderer produces.</p></dd>
 
+ <dt><dfn>User agents that support a full HTML CSS engine</dfn></dt>
+ <dd><p>All processing requirements in this specification apply. However, only a limited set of CSS
+ styles is allowed because <a>user agents that do not support a full HTML CSS engine</a> will need
+ to implement CSS functionality equivalents. User agents that support a full CSS engine must
+ therefore limited the CSS styles they interpret for WebVTT so as to enable identical rendering
```

Comment:
typo: s/limited/limit

Is "interpret" the right term here, addressing your concerns @dwsinger ? Seems to me that "apply" would be better.

Reply:
yes, s/limited/limit/ and "limit the CSS they apply" might be better

Reply:
ok